src/services/user-requests/user-requests.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Service } from './user-requests.class'
+import { ImpressoApplication } from '../../types'
+import { ServiceOptions } from '@feathersjs/feathers'
+import { authenticate } from '@feathersjs/authentication'
+import { queryWithCommonParams } from '../../hooks/params'
+
+const obfuscateProperties = () => (context: any) => {
+  if (context.result) {
+    delete context.result.user_id
+    delete context.result.id
+    context.result = {
+      ...context.result,
+      user_uid: context.params.user.uid,
+    }
+  }
+  return context
+}
+
+export default (app: ImpressoApplication) => {
+  app.use(
+    '/user-requests',
+    new Service({
+      app,
+      name: 'user-requests',
+    }),
+    {
+      events: [],
+    } as ServiceOptions
+  )
+  const service = app.service('user-requests')
+  service.hooks({
+    before: {
+      all: [authenticate('jwt')],
```

Comment:
I suspect this should be locked down only to users belonging to a certain group - those who are allowed to review and approve.

Reply:
The reviewer will have a separate endpoint `user-requests-reviews` user-requests are personal requests, it's not clear then :(